README.md

```diff
@@ -28,6 +28,9 @@ This environment is meant to support Trezor development - both Firmware and Suit
   - Proxies requests to the bridge
 - **Bitcoin regtest**
   - Bitcoind with [blockbook](https://github.com/trezor/blockbook) running in regtest mode
+  - Default credentials for bitcoin backend
+    - user and password: `rpc/rpc` and that you can run
+  - test the connection with `bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc -getinfo`
```

Comment:
Capital `t`. I know it is nit-picking but let's have nice readmes!
You can also add "You can test.."

Reply:
done